include/verilogAST.hpp

```diff
@@ -319,6 +319,15 @@ class BlockComment : public StructuralStatement, public BehavioralStatement {
   ~BlockComment(){};
 };
 
+class InlineVerilog : public StructuralStatement, public BehavioralStatement {
```

Comment:
Is there any expectation/constraints on what `value` can correspond to? Like if I pass in an entire module declaration is that valid? Or only "body" stuff? Presumably this can't checked but it should be documented.

Reply:
I don't think we can validate the contents, but in terms of rules:

It can be used inside the body of a module definition, so any valid statement or collection of statements is valid.  Technically it can also be used inside any always block, but that won't really matter for our current use case (since we'll just be using this to allow inline verilog like assertions inside a magma definition), later on we could also support inline expressions, but again I don't think that would be used in the current intended flow.  For now, I'll change it to just Structural statement and document the intended use case.